uns_plugin/uns_input.go

```
@@ -0,0 +1,178 @@
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/redpanda-data/benthos/v4/public/service"
+	"github.com/twmb/franz-go/pkg/kgo"
+)
+
+// init registers the "uns" batch input plugin with Benthos using its configuration and constructor.
+func init() {
+	service.RegisterBatchInput("uns", RegisterConfigSpec(), newUnsInput)
```

golangci-lint (1.64.8)

Error return value of `service.RegisterBatchInput` is not checked

(errcheck)